VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconDrawable.java

```diff
@@ -1,47 +1,40 @@
 package org.edx.mobile.third_party.iconify;
 
-/**
- * Copyright 2013 Joan Zapata
```

Comment:
The changes here are just copied over from my pull request there: JoanZapata/android-iconify#134

The only divergence we have from the source repo is that we support RTL mirroring of the icons, I [submitted](https://github.com/JoanZapata/android-iconify/pull/86) those changes as well, but they were [reverted](https://github.com/JoanZapata/android-iconify/commit/a628da6cc1d515c831526122740e1b46d5a376a5) when it was upgraded to Font Awesome 4.4.0.